.github/workflows/all-static-checks.yaml

```diff
@@ -50,4 +50,4 @@ jobs:
           if (( $(grep -Rnw 'tests/tt_metal' -e 'tt_lib' | wc -l ) > 0 )); then exit 1; fi
           if (( $(grep -Rnw 'tests/tt_metal' -e 'tt_eager' | wc -l ) > 10 )); then exit 1; fi
       - name: Check ttnn is not used in tt_eager tests more than allowed count
-        run: if (( $(grep -Rnw 'tests/tt_eager' -e 'ttnn' | wc -l ) > 300 )); then exit 1; fi
+        run: if (( $(grep -Rnw 'tests/tt_eager' -e 'ttnn' | wc -l ) > 400 )); then exit 1; fi
```

Comment:
I propose to remove this check.

@tt-rkim

Reply:
@bbradelTT looping you in too

Given the plan to merge the two, I'm assuming the purpose of this check goes away?

Reply:
> @bbradelTT looping you in too
> 
> Given the plan to merge the two, I'm assuming the purpose of this check goes away?

I would expect that, yes.
@ayerofieiev-tt could you please confirm.

Reply:
Confirming

Reply:
I removed the check.